### ISSUE TITLE
regex in app:get/app:post

### ISSUE THREAD

**Issue by yurevich1** [leafo/lapis#242]
Is is possible to use regular expressions as in request string?
for instance, app:get("^theusefullfeature", ...)

**Comment by leafo** [comment]
nope, sorry. Not at the moment. The route parser is very basic and I would like to add some updates to it soon.

**Comment by leafo** [comment]
Another idea from #351 is providing your own router class
